{
  "threat_severity": "Moderate",
  "public_date": "2016-09-25T00:00:00Z",
  "bugzilla": {
    "description": "CVE-2016-7952 libXtst: Insufficient validation of server responses result in various data mishandlings",
    "id": "1381922",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1381922"
  },
  "cvss": {
    "cvss_base_score": "5.4",
    "cvss_scoring_vector": "AV:A/AC:M/Au:N/C:P/I:P/A:P",
    "status": "draft"
  },
  "cvss3": {
    "cvss3_base_score": "5.0",
    "cvss3_scoring_vector": "CVSS:3.0/AV:A/AC:H/PR:N/UI:N/S:U/C:L/I:L/A:L",
    "status": "draft"
  },
  "details": [
    "X.org libXtst before 1.2.3 allows remote X servers to cause a denial of service (infinite loop) via a reply in the (1) XRecordStartOfData, (2) XRecordEndOfData, or (3) XRecordClientDied category without a client sequence and with attached data."
  ],
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 5",
      "fix_state": "Will not fix",
      "package_name": "libXtst",
      "cpe": "cpe:/o:redhat:enterprise_linux:5"
    },
    {
      "product_name": "Red Hat Enterprise Linux 6",
      "fix_state": "Will not fix",
      "package_name": "libXtst",
      "cpe": "cpe:/o:redhat:enterprise_linux:6"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Will not fix",
      "package_name": "libXtst",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    }
  ],
  "upstream_fix": "libXtst 1.2.3",
  "references": [
    "https://lists.x.org/archives/xorg-announce/2016-October/002720.html"
  ],
  "name": "CVE-2016-7952"
}
